### ISSUE TITLE
Feature: registry manager cmd

### ISSUE THREAD

**Issue by cuisongliu** [labring/sealos#3138]
### What is the problem this feature will solve?

sealos manager outside registry

sealos registry list
sealos registry info xxx
sealos registry delete xxxx
sealos reigstry add xxx

add registry info to image-cri-shim config and auto upgrade cri config

TODO List

- [x] #4125
- [ ] image-shim config user password and so on
- [ ] using sealos registry  to registry config  manager

**Comment by kinbod** [comment]
This a delicious feature for sealos。。。。

**Comment by fanux** [comment]
I agree with your opinion and it seems more appropriate to include it in a separate command-line tool for registry CLI.
@kinbod

**Comment by cuisongliu** [comment]
TODO List

- [ ]  all cri container support  insecure-registries config for containerd,docker,crio 
- [ ] image-shim config user password and so on
- [ ] using sealos registry  to registry config  manager
